SceytChatUiKit/src/main/java/com/sceyt/chatuikit/persistence/dao/MessageDao.kt

```diff
@@ -302,6 +309,9 @@ abstract class MessageDao {
     @Query("select message_id as id, tid from messages where channelId =:channelId and message_id <= :id and deliveryStatus in (:status)")
     abstract suspend fun getMessagesTidAndIdLoverThanByStatus(channelId: Long, id: Long, vararg status: DeliveryStatus): List<MessageIdAndTid>
 
+    @Query("select * from AutoDeletedMessages where channelId = :channelId and autoDeleteAt <= :localTime")
```

Comment:
I have asked to create separate DAO for AutoDeletedMessages. Please remove thsi from MessageDao and create separate AutoDeleteMessagesDao.

Also i thins AutoDeleteMessages better than AutoDeletedMessages